README.md

```diff
@@ -76,6 +78,7 @@ If you want to contribute code, you must first read [our contributing guidelines
 - [iron-fish/website-testnet](https://github.com/iron-fish/website-testnet): The repo that powers [testnet.ironfish.network](https://testnet.ironfish.network)
 - [iron-fish/ironfish-api](https://github.com/iron-fish/ironfish-api): The repository that powers most Iron Fish API services.
 - [iron-fish/chain-explorer](https://github.com/iron-fish/chain-explorer): A visual tool to explore the block chain and all of its forks.
+- [iron-fish/block-explorer](https://github.com/iron-fish/block-explorer): V2 of chain explorer.
```

Comment:
Is this a correct description?

Reply:
What do you think about, "The repo that powers our block explorer at <link>"? Like the first entry?